src/watcher.js

```diff
@@ -213,7 +213,7 @@ module.exports = class MonoplasmaWatcher extends EventEmitter {
 
     async stop() {
         //this.tokenFilter.unsubscribe()
-        this.channel.close()
+        await this.channel.close()
```

Comment:
good catches. I wonder, I guess neither lint nor IDE warn about unhandled promises? TypeScript would, you have to add a `void` in front (also valid JS!) in order to say "it's ok, I'm not going to await this promise".